src-web/components/DefaultLayout.tsx

```diff
@@ -16,17 +15,14 @@ export function DefaultLayout() {
           <Toasts />
           <Dialogs />
         </>
-        <motion.div
-          initial={{ opacity: 0 }}
-          animate={{ opacity: 1 }}
-          transition={{ duration: 0.1, delay: 0.1 }}
```

Comment:
Was getting stuck at `0` opacity because of the lazy route loading that was added recently (I think that's the issue anyway). So we'l just remove it